diff --git a/node_modules/@strapi/admin/admin/src/pages/AuthPage/components/Register/index.js b/node_modules/@strapi/admin/admin/src/pages/AuthPage/components/Register/index.js
index fea4734..8eb427f 100644
--- a/node_modules/@strapi/admin/admin/src/pages/AuthPage/components/Register/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/AuthPage/components/Register/index.js
@@ -177,7 +177,7 @@ const Register = ({ authType, fieldsToDisable, noSignin, onSubmit, schema }) =>
                       {formatMessage({
                         id: 'Auth.form.register.subtitle',
                         defaultMessage:
-                          'Credentials are only used to authenticate in Strapi. All saved data will be stored in your database.',
+                          'Credentiaddls are only used to authenticate in Strapi. All saved data will be stored in your database.',
                       })}
                     </Typography>
                   </Flex>
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
index 84ec0ac..1662592 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
@@ -22,83 +22,7 @@ const ContentBlocks = () => {
 
   return (
     <Flex direction="column" alignItems="stretch" gap={5}>
-      <CloudBox />
-      <BlockLink
-        href="https://strapi.io/resource-center"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonReadTheDocumentationSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'global.documentation',
-            defaultMessage: 'Documentation',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.documentation.content',
-            defaultMessage: 'Discover the essential concepts, guides and instructions.',
-          })}
-          icon={<InformationSquare />}
-          iconBackground="primary100"
-        />
-      </BlockLink>
-      <BlockLink
-        href="https://strapi.io/starters"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonCodeExampleSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.code',
-            defaultMessage: 'Code example',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.code.content',
-            defaultMessage: 'Learn by using ready-made starters for your projects.',
-          })}
-          icon={<CodeSquare />}
-          iconBackground="warning100"
-        />
-      </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog/categories/tutorials"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonTutorialSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.tutorial',
-            defaultMessage: 'Tutorials',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.tutorial.content',
-            defaultMessage: 'Follow step-by-step instructions to use and customize Strapi.',
-          })}
-          icon={<PlaySquare />}
-          iconBackground="secondary100"
-        />
-      </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonBlogSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.blog',
-            defaultMessage: 'Blog',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.blog.content',
-            defaultMessage: 'Read the latest news about Strapi and the ecosystem.',
-          })}
-          icon={<FeatherSquare />}
-          iconBackground="alternative100"
-        />
-      </BlockLink>
+      <p> Welcome to copyright knowledge...</p>
     </Flex>
   );
 };
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
index 8fb94c5..55cbd3b 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
@@ -21,15 +21,16 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
           <Typography as="h1" variant="alpha">
             {hasCreatedContentType
               ? formatMessage({
-                  id: 'app.components.HomePage.welcome.again',
-                  defaultMessage: 'Welcome 👋',
+                  id: 'app.components.HomePage.welcome.agai1n',
+                  defaultMessage: 'Welc1ome 👋',
                 })
               : formatMessage({
                   id: 'app.components.HomePage.welcome',
-                  defaultMessage: 'Welcome on board!',
+                  defaultMessage: 'Welco2me on board!',
                 })}
           </Typography>
           <WordWrap textColor="neutral600" variant="epsilon">
+          test
             {hasCreatedContentType
               ? formatMessage({
                   id: 'app.components.HomePage.welcomeBlock.content.again',
@@ -42,21 +43,6 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
                     'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
                 })}
           </WordWrap>
-          {hasCreatedContentType ? (
-            <Link isExternal href="https://strapi.io/blog">
-              {formatMessage({
-                id: 'app.components.HomePage.button.blog',
-                defaultMessage: 'See more on the blog',
-              })}
-            </Link>
-          ) : (
-            <Button size="L" onClick={onCreateCT} endIcon={<ArrowRight />}>
-              {formatMessage({
-                id: 'app.components.HomePage.create',
-                defaultMessage: 'Create your first Content type',
-              })}
-            </Button>
-          )}
         </Flex>
       </Box>
     </div>
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
index e290ab2..a1d3c57 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
@@ -135,7 +135,7 @@ const SocialLinks = () => {
   const { communityEdition } = useAppInfo();
 
   const socialLinksExtended = [
-    ...socialLinks,
+//    ...socialLinks,
     {
       icon: <StyledStrapi />,
       link: communityEdition
@@ -143,7 +143,17 @@ const SocialLinks = () => {
         : 'https://support.strapi.io/support/home',
       name: {
         id: 'Settings.application.get-help',
-        defaultMessage: 'Get help',
+        defaultMessage: 'Quicklink 1',
+      },
+    },
+    {
+      icon: <StyledStrapi />,
+      link: communityEdition
+        ? 'https://discord.strapi.io'
+        : 'https://support.strapi.io/support/home',
+      name: {
+        id: 'Settings.application.get-help',
+        defaultMessage: 'Quicklink 2',
       },
     },
   ];
@@ -160,36 +170,11 @@ const SocialLinks = () => {
       paddingBottom={6}
       shadow="tableShadow"
     >
-      <Box paddingBottom={7}>
-        <Flex direction="column" alignItems="stretch" gap={5}>
-          <Flex direction="column" alignItems="stretch" gap={3}>
-            <Typography variant="delta" as="h2" id="join-the-community">
-              {formatMessage({
-                id: 'app.components.HomePage.community',
-                defaultMessage: 'Join the community',
-              })}
-            </Typography>
-            <Typography textColor="neutral600">
-              {formatMessage({
-                id: 'app.components.HomePage.community.content',
-                defaultMessage:
-                  'Discuss with team members, contributors and developers on different channels',
-              })}
-            </Typography>
-          </Flex>
-          <Link href="https://feedback.strapi.io/" isExternal endIcon={<ExternalLink />}>
-            {formatMessage({
-              id: 'app.components.HomePage.roadmap',
-              defaultMessage: 'See our road map',
-            })}
-          </Link>
-        </Flex>
-      </Box>
       <GridGap>
         {socialLinksExtended.map(({ icon, link, name }) => {
           return (
             <GridItem col={6} s={12} key={name.id}>
-              <LinkCustom size="L" startIcon={icon} variant="tertiary" href={link} isExternal>
+              <LinkCustom size="L" variant="tertiary" href={link} isExternal>
                 {formatMessage(name)}
               </LinkCustom>
             </GridItem>
